scripts/runTest.js

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env node
+
+/**
+ * @file Script to validate JSON files against given tests
+ *
+ * Usage: node <script-path>.js json/file/path.json mandatoryTest_6_1_1
+ *
+ * `mandatoryTest_6_1_1` is a sample here. You can insert any test name from lib/mandatoryTests.js,
+ *  `lib/optionalTests.js`, `lib/schemaTests.js` and `lib/schemaTests.js`.
+ */
+
+import { readFile } from 'fs/promises'
+import * as schemaTests from '../schemaTests.js'
+import * as mandatoryTests from '../mandatoryTests.js'
+import * as optionalTests from '../optionalTests.js'
+import * as informativeTests from '../informativeTests.js'
+import validate from '../validate.js'
+
+const [, , filePath, testName] = process.argv
+
+const json = JSON.parse(await readFile(filePath, { encoding: 'utf-8' }))
+const tests =
+  /** @type {Array<import('../lib/shared/types.js').DocumentTest>} */ (
+    Object.values(mandatoryTests)
+  )
+    .concat(Object.values(optionalTests))
+    .concat(Object.values(informativeTests))
+    .concat(Object.values(schemaTests))
+
+const matchingTests = tests.filter((t) => t.name === testName)
```

Comment:
Maybe, we should allow also presets? e.g. (just throwing some quick and dirty code in):
```suggestion
var matchingTests = tests.filter((t) => t.name === testName)

switch (testName) {
   case 'mandatory':
           matchingTests = Object.values(mandatoryTests);
           break;
   case 'optional':
           matchingTests = Object.values(optionalTests);
           break;
     case 'informative':
           matchingTests = Object.values(informativeTests);
           break;
     case 'schema':
           matchingTests = Object.values(schemaTests);
           break;
      case 'valid':
           matchingTests = Object.values(schemaTests).concat(Object.values(mandatoryTests));
           break;
      default:
           // Just use the one from line 30
 }